.travis.yml

```diff
@@ -14,40 +14,32 @@ env:
     - PYTHON_VERSION=3.3
     - PYTHON_VERSION=3.4
   global:
-      - SETUPTOOLS_VERSION=stable
+    - SETUPTOOLS_VERSION=stable
+    - CONDA_DEPENDENCIES="pytest<2.6 sphinx cython numpy"
+    - PIP_DEPENEDENCIES="coveralls pytest-cov"
 
 matrix:
     include:
         - os: linux
           env: PYTHON_VERSION=2.7 SETUPTOOLS_VERSION=dev
+               CONDA_DEPENDENCIES="mercurial mock"
 
 before_install:
 
-    # Use utf8 encoding. Should be default, but this is insurance against
-    # future changes
-    - export PYTHONIOENCODING=UTF8
+    - if [[ $TRAVIS_OS_NAME == linux ]]; then apt-get update; apt-get install graphviz; fi
```

Comment:
Is there a reason ``astropy-helpers`` is not tested on the container travis? That case ``graphviz`` could be installed there, isn't it?